Improving the license workflow acceptance documentation

Notes from working on #993.

The doc in question: https://github.com/spdx/license-list-XML/blob/master/DOCS/new-license-workflow.md#if-license-is-accepted

Reviewing 'If license is accepted'

- [x] Link `SPDX Online tools` in 7.i.
    * https://spdxtools.sourceauditor.com/
- [ ] 7.i assumes that the license exists in the License Requests list
    * In the case of #767, the CC-BY-NC-ND-3.0-IGO license isn't in the list.
- [x] 7.i.a could use a link to a "wrapped text version" example.
- [x] 7.i "edit XML" requires a login
    * Mention this in the doc
    * Only signin option is 'Login with GitHub'
- [x] 7.i "edit XML" is a button at the bottom of the license text (License Requests > click license > scroll to the bottom
    * This is NOT the 'License XML Editor' link from the top of the page
- [x] 7.i.b should include directions for where `listVersionAdded=` should be added, and perhaps an example same.
- [ ] 7.i.c says "standard license header" as though it's common knowledge. It's not. An example is required.
    * I don't know where to find this or to what it refers.
- [ ] 7.i.d has a broken link to an Overview page.
- [x] 7.i.f should link to a license to use as an example. It also should explain "the bullet or list tags" better. Where would these tags be found?
- [x] 7.ii says to run the tests locally, but the changes are happening using the online tool. There has to be a "copy your changes locally, then…" step in there.
    * Also, at no point was the user directed to clone the repo locally. Probably should ask them to do that? But if they're doing all the work in the online tool, maybe just drop the local test step?

Example text from an error:

```
steve@clariel: license-list-XML> ./test-one-license BSD-3-Clause
gpg --verify --no-default-keyring --keyring ./goneall.gpg licenseListPublisher-2.1.22.jar.asc
gpg: assuming signed data in 'licenseListPublisher-2.1.22.jar'
gpg: Signature made Tue Oct 15 21:58:59 2019 EDT
gpg:                using RSA key 8CCBE63DE91BFF1C
gpg: Good signature from "Gary O'Neall (Key for signing jar files) <gary@sourceauditor.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: EA76 0AF7 CBD0 E4F9 013F  06FB 8CCB E63D E91B FF1C
java -jar -DLocalFsfFreeJson=false -DlistedLicenseSchema="schema/ListedLicense.xsd" licenseListPublisher-2.1.22.jar LicenseRDFAGenerator src '.tmp' 1.0 2000-01-01 test/simpleTestForGenerator expected-warnings
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console. Set system property 'log4j2.debug' to show Log4j2 internal initialization logging.
Processing License List.
Processing Exceptions
Writing table of contents
The following warning(s) were identified:
	Test for license BSD-3-Clause failed: Test for license ID BSD-3-Clause failed due to difference found Normal text of license does not match starting at line #11 column #26 "by" when comparing to template text " be used to endorse or promote products derived from this software without specific prior written pe..."
Completed processing licenses
make: *** [validate-canonical-match] Error 64
```

--- a/DOCS/new-license-workflow.md
+++ b/DOCS/new-license-workflow.md
@@ -4,7 +4,7 @@
 
 This document provides guidance and checklists for SPDX legal team members who are assigned to shepherd a new license request.
 
-The instructions here assume the requestor has already submitted the new license/exception request via the [SPDX online tools](http://13.57.134.254/app/).
+The instructions here assume the requestor has already submitted the new license/exception request via the [SPDX online tools](http://13.57.134.254/app/). Note that it is also possible for license requests to be submitted directly as Issues in the GitHub repo; when this occurs, they will not appear on the SPDX online tools page.
 
 New license requests are to be assigned to an SPDX legal team member and that person is responsible for following the request through to its final determination.  Make sure to add yourself as the Assignee in the license request issue here on GitHub, add the [label](https://github.com/spdx/license-list-XML/labels) `new license/exception request`, and add the appropriate release milestone to the issue.
 
@@ -17,8 +17,9 @@ New license requests are to be assigned to an SPDX legal team member and that pe
     3. If additional markup can accommodate the license, then the license does not need to be added: inform the requestor, comment on the issue as such, then create a PR for the existing license with the additional markup, and close issue once the PR has been merged.
 
 2. If the submitter is not the license author or steward, ask for that contact or try to find that person or organization to make them aware the license has been submitted.
-3. Check the submission for any other missing information, e.g., working URL, examples of use, full text, standard header, etc. See the Field definitions on Overview page for assistance.
+3. Check the submission for any other missing information, e.g., working URL, examples of use, full text, standard header, etc. See the Field definitions on [Overview page](https://spdx.dev/license-list/) for assistance.
 Ask the submitter for any additional info needed, preferably via the Github issue, if possible. Record any updates there.
+    1. The "standard header" or "official license header" is defined in the [SPDX Matching Guidelines (sec. 1.1.1)](https://spdx.dev/license-list/matching-guidelines/) as "specific text specified within the license itself to be put in the header of files."
 4. Review the following, bring any questions to legal team:
     1. Is this an open source license?
     2. Is the short identifier unique? Does this license have a short identifier that is used elsewhere already (e.g., Fedora)?
@@ -173,7 +174,7 @@ The [SPDX Online tools](http://13.57.134.254/app/license_requests/) are an optio
     1. The current XML output does not implement some of XML tagging and may mark every new line with a paragraph tag depending on input. If this has happened, it may be more efficient to re-submit the license text, using a wrapped text version.
     1. Make sure to include listVersionAdded= and the correct license list version number for the upcoming release
     1. Check for a standard license header
-    1. Check if there should be any Notes based on Notes field descripiton in [link](Overview page)
+    1. Check if there should be any Notes based on Notes field descripiton in the [Overview page](https://spdx.dev/license-list/)
     1. Check that we have a working URL for the license text in the wild. If using a link in GitHub, include a link to a specific commit
     1. Check all the XML formatting: the current XML output does not insert the bullet or list tags. Use a previously submitted license as reference for how to format
     1. If you have questions about text that could be optional or could be replaceable, add a comment to the PR, once made or add a reviewer to check it

